aws/ami/files/scylla_install_ami

```diff
@@ -60,64 +68,106 @@ if __name__ == '__main__':
         print('Error: need to specify --localrpm or --repo/--repo-for-install')
         sys.exit(1)
 
-    run('yum update -y')
+    if distro == 'centos':
+        run('yum update -y')
+
+        if args.repo_for_install:
+            run('curl -L -o /etc/yum.repos.d/scylla_install.repo {REPO_FOR_INSTALL}'.format(REPO_FOR_INSTALL=args.repo_for_install))
 
-    if args.repo_for_install:
-        run('curl -L -o /etc/yum.repos.d/scylla_install.repo {REPO_FOR_INSTALL}'.format(REPO_FOR_INSTALL=args.repo_for_install))
+        if args.localrpm:
+            rpms = glob.glob('{}/scylla*.*.rpm'.format(homedir))
+            run('yum install -y {}'.format(' '.join(rpms)))
+        else:
+            run('yum install -y {0}-machine-image {0}-debuginfo'.format(args.product))
 
-    if args.localrpm:
-        rpms = glob.glob('{}/scylla*.*.rpm'.format(homedir))
-        run('yum install -y {}'.format(' '.join(rpms)))
+        if args.repo_for_install:
+            os.remove('/etc/yum.repos.d/scylla_install.repo')
+        if args.repo_for_update:
+            run('curl -L -o /etc/yum.repos.d/scylla.repo {REPO_FOR_UPDATE}'.format(REPO_FOR_UPDATE=args.repo_for_update))
     else:
-        run('yum install -y {0}-machine-image {0}-debuginfo'.format(args.product))
+        run('apt-get update')
```

Comment:
Need to add a key before update
```suggestion
        run('sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 5e08fbd8b5d6ec9c ')
        run('apt-get update')
```

Reply:
> Need to add a key before update

Added `apt-key` for scylla_install_ami, too.